src/forum-tool-selector-settings/ForumToolSelectorSettings.jsx

```diff
@@ -0,0 +1,130 @@
+    supportLevel: 'Full support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'Piazza1',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Piazza is designed to connect students, TAs, and professors so every student can get the help they need when they need it',
+    supportLevel: 'Partial support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'Yellowdig1',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Yellowdig is the digital solution that impacts the entire student lifecycle and enables lifelong learning.',
+    supportLevel: 'Coming soon',
+    isAvailable: false,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'edX Forum2',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Start conversations with other learners, ask questions, and interact with other learners in the course.',
+    supportLevel: 'Full support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+  {
+    forumId: 'Piazza2',
+    logo: 'https://cdn-blog.lawrencemcdaniel.com/wp-content/uploads/2018/01/22125436/edx-logo.png',
+    description: 'Piazza is designed to connect students, TAs, and professors so every student can get the help they need when they need it',
+    supportLevel: 'Partial support',
+    isAvailable: true,
+    features: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'WCAG 2.1 Support'],
+    allfeatures: ['LTI Integration', 'Discussion Page', 'Embedded Course Sections', 'Embedded Course Units', 'WCAG 2.1 Support'],
+  },
+];
+
+function chunk(arr, len) {
+  const chunks = [];
+  let i = 0;
+  const n = arr.length;
+  while (i < n) {
+    chunks.push(arr.slice(i, i += len));
+  }
+  return chunks;
+}
+
+function ForumToolSelectorSettings({ intl }) {
+  const [selectedForumId, setSelectedForumId] = useState(null);
+
+  const onSelect = (forumId) => {
+    setSelectedForumId(forumId);
+  };
```

Comment:
Remove this, and make it external, i.e. pass in a selectedForumId, and a onSelectForum function.